docs/index.md

```diff
@@ -1,30 +1,44 @@
 ---
-page_title: "Provider: AnxCloud"
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "anxcloud Provider"
 subcategory: ""
 description: |-
-  Terraform provider for interacting with Anexia Cloud API.
+  
 ---
 
-# AnxCloud Provider
+# anxcloud Provider
 
--> Visit the [Anexia official website](https://anexia.com/en/) to get more info about Anexia Cloud.
 
-The AnxCloud provider is used to interact with Anexia Cloud API.
```

Comment:
Is this moved somewhere else or just completely lost?

Reply:
I've added it manually.